eng/versioning/version_client.txt

```diff
@@ -52,10 +52,10 @@ com.azure:azure-analytics-synapse-spark;1.0.0-beta.3;1.0.0-beta.4
 com.azure:azure-analytics-synapse-managedprivateendpoints;1.0.0-beta.3;1.0.0-beta.4
 com.azure:azure-analytics-synapse-monitoring;1.0.0-beta.3;1.0.0-beta.4
 com.azure:azure-communication-chat;1.0.0;1.1.0-beta.1
-com.azure:azure-communication-common;1.0.1;1.1.0-beta.1
-com.azure:azure-communication-sms;1.0.1;1.1.0-beta.1
-com.azure:azure-communication-identity;1.1.0;1.2.0-beta.1
-com.azure:azure-communication-phonenumbers;1.0.1;1.1.0-beta.1
+com.azure:azure-communication-common;1.0.2;1.0.2
+com.azure:azure-communication-sms;1.0.1;1.0.3
+com.azure:azure-communication-identity;1.1.0;1.1.1
+com.azure:azure-communication-phonenumbers;1.0.1;1.0.3
```

Comment:
Shouldn't sms/phonenumbers be 1.0.2;1.0.3?

Reply:
I think phone number can remain 1.0.1;1.0.3